src/main/java/com/wrapper/Model/CareContextTable.java

```diff
@@ -0,0 +1,162 @@
+//
+// Source code recreated from a .class file by IntelliJ IDEA
+// (powered by FernFlower decompiler)
+//
+
+package com.wrapper.Model;
```

Comment:
lets create a package named mongo.tables and have these classes be created under it